package.json

```diff
@@ -12,25 +12,25 @@
     "console": "npx truffle console",
     "compile": "npm run compile-contracts",
     "compile-requests": "npx rad2sol",
-    "compile-contracts": "npm run postinstall && npx truffle compile",
+    "compile-contracts": "npm run postinstall && npm run compile-requests && npx truffle compile",
     "compile-flattened": "npx truffle compile --all --config flattened-config.js ",
     "coverage": "solidity-coverage",
-    "flatten": " mkdir contracts/flattened/ 2>/dev/null; npx truffle-flattener contracts/*/*.sol > contracts/flattened/Flattened.sol",
+    "flatten": "node ./scripts/flatten.js 2>&1",
     "fmt": "solhint --max-warnings 0 \"contracts/**/*.sol\" && eslint ./test && solhint \"test/**/*.sol\"",
     "fmt!": "solhint --max-warnings 0 \"contracts/**/*.sol\" --fix && eslint ./test --fix && solhint \"test/**/*.sol\" --fix",
-    "postinstall": "mkdir -p build/contracts >/dev/null 2>&1; cp node_modules/witnet-ethereum-bridge/build/contracts/WitnetRequestBoard*.json build/contracts",
+    "postinstall": "node ./scripts/postinstall.js 2>&1",
     "solium": "solium -d contracts",
     "solium:fix": "solium -d contracts --fix",
     "migrate": "npx truffle migrate",
-    "migrate-flattened": "npm run flatten && npx truffle migrate --config flattened-config.js",
-    "verify-flattened": "npx truffle run verify"
+    "migrate-flattened": "npm run compile-flattened && npx truffle migrate --config flattened-config.js",
+    "verify-flattened": "npx truffle run verify --config flattened-config.js"
   },
   "dependencies": {
     "@openzeppelin/contracts": "^4.1.0",
-    "adomedianizer": "git+https://github.com/adoracles/ado-contracts",
+    "adomedianizer": "https://github.com/adoracles/ado-contracts",
```

Comment:
Why did you remove the `git` protocol?

I just discovered that according to the npm documentation, GitHub repositories we could be referred as:
```
"foo": "user/foo-project"`
```

Reply:
Forcing importing latest version on install.